### ISSUE TITLE
Building using Docker fails due to infinite recursion (ENAMETOOLONG)

### ISSUE THREAD

**Issue by mrothenbuecher** [electron-userland/electron-packager#548]
The build for windows fails in the gitlab CI.
If I run the command on a windows machine everything runs fine.

```
image: node:6.9.2
install:
  script:
   - apt-get update
   - apt-get install -y wine
   - npm install electron-packager -g
   - npm install
   - npm run build
```

```
...
Downloading electron-v1.4.13-win32-ia32.zip
[============================================>] 100.0% of 44.03 MB (35.22 MB/s)
Packaging app for platform win32 ia32 using electron v1.4.13
ENAMETOOLONG: name too long, unlink '/builds/mrothenbuecher/5ld/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/electron-packager/win32-ia32/5ld-win32-ia32/resources/app/node_modules/unreachable-branch-transform/unreachableBranchTransformer.js'

npm info lifecycle 5ld@0.0.1~build: Failed to exec build script
npm ERR! Linux 4.7.3-coreos-r3
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "build"
npm ERR! node v6.9.2
npm ERR! npm  v3.10.9
npm ERR! code ELIFECYCLE
npm ERR! 5ld@0.0.1 build: `rimraf *db && rimraf dist && electron-packager . 5ld --platform=win32 --arch=all --overwrite --out=dist --icon=images/wot && cpr portlets dist/5ld-win32-ia32/portlets && cpr portlets dist/5ld-win32-x64/portlets`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the 5ld@0.0.1 build script 'rimraf *db && rimraf dist && electron-packager . 5ld --platform=win32 --arch=all --overwrite --out=dist --icon=images/wot && cpr portlets dist/5ld-win32-ia32/portlets && cpr portlets dist/5ld-win32-x64/portlets'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the 5ld package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     rimraf *db && rimraf dist && electron-packager . 5ld --platform=win32 --arch=all --overwrite --out=dist --icon=images/wot && cpr portlets dist/5ld-win32-ia32/portlets && cpr portlets dist/5ld-win32-x64/portlets
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs 5ld
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls 5ld
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /builds/mrothenbuecher/5ld/npm-debug.log
ERROR: Build failed: exit code 1
```

I am using the following arguments

```
electron-packager . 5ld --platform=win32 --arch=all --overwrite --out=dist --icon=images/wot
```

**Comment by MarshallOfSound** [comment]
You almost certainly need to use the Docker / strange linux agent work around that I use here

https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/blob/master/gulpfile.babel.js#L58

**Comment by malept** [comment]
There's a really long thread about it here: https://github.com/electron-userland/electron-packager/issues/396

TL;DR: I know about the bug, I just don't know about a good way of fixing it. I feel like just adding `/electron-packager/` to the default ignore list is a bit heavy-handed.
